model_cloud/cloud_api.py

```diff
@@ -0,0 +1,79 @@
+import json
```

Comment:
Let's move this under lightning_diffusion as `lightning_diffusion/model_cloud/cloud_api.py`